Pre-Release 3.1.0-10

christoph-morrison opened the pull request:
# Pre-Release 3.1.0-10

## ⚠️ Important
* Perl version 5.26 is now the minimum version (#321)

## Content

### Bugs
* #324 Typo in Length
* #325 `&region` evaluates to `®ion`
* #327  Fixed double `return` statement
* #131 fixed startsIn if it's rainin right now reporting nothing back
* Fixed some typos

### Enhancements
* #326 Refactored the URL building
* #321 Perl > 5.26 is now mandatory
* #330 Importing POXIS without restrictions is not encouraged
* #332 Refactored debug messages
* #312 Support FHEMs `$init_done`
* #130 Added reading `rainBeginUnixtime` with the next rain as Unix timestamp (or -1 if no precipitation is predicted)
* Removed some unnecessarily defined variables

christoph-morrison committed:
#326 refactored request url build, workaround for #325, but not an adequate solution

actions-user committed:
Automatic updated controls and CHANGED

christoph-morrison committed:
#324 fixed typo in $ARGUMENT_LENGHT_WITH_LOC

actions-user committed:
Automatic updated controls and CHANGED

christoph-morrison committed:
#327 fixed double return

actions-user committed:
Automatic updated controls and CHANGED

christoph-morrison committed:
#131 fixed startsIn if it's rainin right now reporting nothing back

actions-user committed:
Automatic updated controls and CHANGED

christoph-morrison committed:
Unused variable language removed

christoph-morrison committed:
Unused variable $fhemweb_name removed

christoph-morrison committed:
Unused variable $arg removed

christoph-morrison committed:
Unused variable @args removed

christoph-morrison committed:
Unused variable $room removed

christoph-morrison committed:
Unused variable $page_definition removed

christoph-morrison committed:
Typo in $LAMETRIC_MULTIPLIER

christoph-morrison committed:
Unused variable $hash removed

christoph-morrison committed:
#326 removed parentheses from sprintf

christoph-morrison committed:
Removed some now obsolete comments

christoph-morrison committed:
#321 updated meta.json also

christoph-morrison committed:
Bump to v3.0.10

christoph-morrison committed:
#321 updated comments

actions-user committed:
Automatic updated controls and CHANGED

christoph-morrison committed:
#326 #325 fix for the fix

actions-user committed:
Automatic updated controls and CHANGED

christoph-morrison committed:
Fixed typos in $INTERVAL_LENGTH_MINUTES and $INTERVAL_LENGTH_SECONDS

christoph-morrison committed:
#326 #325 fix for the fix for the fix

actions-user committed:
Automatic updated controls and CHANGED

christoph-morrison committed:
#330 disabled automatic exports for POSIX

christoph-morrison committed:
Bump to semantic version 3.1.0-10

actions-user committed:
Automatic updated controls and CHANGED

christoph-morrison committed:
#332 Be less verbose

actions-user committed:
Automatic updated controls and CHANGED

christoph-morrison committed:
#332 refactored debug_message

christoph-morrison committed:
Bump to 3.1.0-11

christoph-morrison committed:
#312 consider $init_done

christoph-morrison committed:
Use docker, update vagrant to bullseye

actions-user committed:
Automatic updated controls and CHANGED

christoph-morrison committed:
#130 rainBeginUnixtime added, delivers the next precipitation as unix timestamp or -1 if no precipitation is predicted

actions-user committed:
Automatic updated controls and CHANGED

christoph-morrison committed:
#329 #328 added i18n for starts in

actions-user committed:
Automatic updated controls and CHANGED

christoph-morrison committed:
Back to 3.1.0-10

actions-user committed:
Automatic updated controls and CHANGED

christoph-morrison committed:
#333 #328 Added sanity checks for latitude and longitude

actions-user committed:
Automatic updated controls and CHANGED

christoph-morrison commented on controls_Buienradar.txt:
```diff
@@ -1,4 +1,4 @@
-UPD 2021-09-30_13:40:26 58237 lib/FHEM/Weather/Buienradar.pm
+UPD 2021-10-14_12:24:16 61658 lib/FHEM/Weather/Buienradar.pm
 DEL FHEM/59_Buienradar.pm
-UPD 2021-09-30_13:40:26 13538 FHEM/59_Buienradar.pm
```
Broken, should be 13537

sidey79 replied:
FHEM/59_Buienradar.pm seems not to be edited in the commit before.

christoph-morrison replied:
Nope, it's just a very simple wrapper. Don't know why wrong a size ist calculated, again. Locally, it's also 13537.

christoph-morrison committed:
#335 Manually fixed the size for 59_Buienradar.pm

#335 Manually fixed the size for 59_Buienradar.pm for a spot on fix for users who can't update anymore

actions-user committed:
Automatic updated controls and CHANGED

christoph-morrison committed:
#335 Disabled documentation

actions-user committed:
Automatic updated controls and CHANGED
